app/Resources/assets/js/jquery.instantSearch.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * jQuery plugin for an instant searching.
+ *
+ * @author Oleg Voronkovich <oleg-voronkovich@yandex.ru>
+ */
+(function($) {
+    $.fn.instantSearch = function(config) {
+        return this.each(function() {
+            initInstantSearch(this, $.extend(true, defaultConfig, config || {}));
+        });
+    };
+
+    var defaultConfig = {
+        allowedTags: '',
+        minQueryLength: 2,
+        maxPreviewItems: 10,
+        previewDelay: 500,
+        noItemsFoundMessage: 'No items found'
+    };
+
+    function debounce(fn, delay) {
+        var timer = null;
+        return function () {
+            var context = this, args = arguments;
+            clearTimeout(timer);
+            timer = setTimeout(function () {
+                fn.apply(context, args);
+            }, delay);
+        };
+    }
+
+    function isValidUrl(url) {
+        var parser = document.createElement('a');
+        try {
+            parser.href = url;
+            return !!parser.hostname;
+        } catch (e) {
+            return false;
+        }
+    }
+
+    // See http://phpjs.org/functions/strip_tags/
+    function stripTags(input, allowed) {
```

Comment:
Don't strip tags. This is not the right way to escape HTML.
And it produces a broken output (if a post title contains a ``<``, it should be displayed, not removed).